laba5.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#include <string>
+#include <vector>
+ 
+enum lol {
+    Isotype,
+    Radioactive,
+    Ion,
+    Antimatter,
+    Stable,
+};
+ 
+class Atom
+{
+public:
+    std::string name;
+    int atomic_mass_unit;
+    int protons_number;
+    int electrons_number;
+    lol type;
+    bool IsNeutral();
+ 
+Atom() {
+    name = "Hydrogen";
+    atomic_mass_unit = 1;
+    protons_number = 1;
+    electrons_number = 1;
+    lol type = Stable;
+}
+ 
+Atom(std::string a, int b, int c, int d, lol e) {
+    name = a;
+    atomic_mass_unit = b;
+    protons_number = c;
+    electrons_number = d;
+    type = e;
+}
+ 
+};
+ 
+bool Atom::IsNeutral() {
+    if (Atom::protons_number == Atom::electrons_number) {
+        return true;
+    } else {
+        return false;
+    }
+}
+ 
+ 
+ 
+ 
+class Molecule
+{
```

Comment:
your code must be wrapped in class by 1 tab or 4 spaces

Reply:
No.